sql/mob_groups.sql

```diff
@@ -3271,6 +3271,7 @@ INSERT INTO `mob_groups` VALUES (14,6957,67,'Mamool_Ja_Profligate',0,128,0,0,0,9
 INSERT INTO `mob_groups` VALUES (15,6958,67,'Mamool_Ja_Unseen',0,128,0,0,0,99,99,0);
 INSERT INTO `mob_groups` VALUES (16,6959,67,'Mamool_Ja_Flamerearer',0,128,0,0,0,99,99,0);
 INSERT INTO `mob_groups` VALUES (17,6960,67,'Scorched-snout_Wyvern',0,128,0,0,0,99,99,0);
+INSERT INTO `mob_groups` VALUES (18,3128,67,'Phantom_Puk',0,128,0,650,0,75,75,0);
```

Comment:
[17051655] = {['id']=17051655, ['name']="PhantomPuk", ['index']=7, ['level']=76},
    [17051656] = {['id']=17051656, ['name']="PhantomPuk", ['index']=8, ['level']=76},
    [17051657] = {['id']=17051657, ['name']="PhantomPuk", ['index']=9, ['level']=76},
    [17051658] = {['id']=17051658, ['name']="PhantomPuk", ['index']=10, ['level']=76},
    [17051659] = {['id']=17051659, ['name']="PhantomPuk", ['index']=11, ['level']=76},

Widescan seems to think these are all 76 (but I think that widescan is not perfectly accurate)

Reply:
fixed